script/download-node.js

```diff
@@ -37,8 +37,14 @@ var downloadTarballAndExtract = function(url, location, callback) {
   });
   stream.on('end', callback.bind(this, tempPath));
   stream.on('error', callback);
-  request.createReadStream({url: url}, function(requestStream) {
-    requestStream.pipe(zlib.createGunzip()).pipe(stream);
+  request.createReadStream({url: url}, function(res) {
```

Comment:
Agreed. I will redo it. Is there a better way to get feedback on a patch without doing a pull request?

Thanks,
Tom

> On May 7, 2015, at 1:14 PM, Kevin Sawicki <notifications@github.com> wrote:
> Mind keeping the variable as requestStream, having both res and resp variables is a little confusing.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/atom/apm/pull/360/files#r29873471>.
>